SetUserSessionLockActivity/index.ts

```diff
@@ -0,0 +1,18 @@
+﻿import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import nodeFetch from "node-fetch";
+import { Client, createClient } from "../utils/sessionApiClient";
+import { createSetUserSessionLockActivityHandler } from "./handler";
+
+const sessionApiUrl = getRequiredStringEnv("SESSION_API_URL");
+const sessionApiKey = getRequiredStringEnv("SESSION_API_KEY");
+
+const client: Client<"ApiKey"> = createClient({
+  baseUrl: sessionApiUrl,
+  fetchApi: (nodeFetch as unknown) as typeof fetch,
```

Comment:
we must use a fetch with timeout like in https://github.com/pagopa/io-functions-admin/blob/master/SendUserDataDownloadMessageActivity/index.ts#L25